app/controllers/conferences_controller.rb

```diff
@@ -16,12 +16,11 @@ def show
 
   def create
     team = current_user.student_team
```

Comment:
Yeah, makes sense just team's student create new conferences. However, in the description of this issue, it was asked to let orga members do it too. Even in the new issue #825 related to add new button, was asked to allow both orga members and students to add new conference.